experimenter/experimenter/experiments/models.py

```diff
@@ -822,6 +822,26 @@ def feature_has_live_multifeature_experiments(self):
             )
         return matching
 
+    @property
+    def excluded_live_deliveries(self):
+        """Live experiments that are excluded by the current experiment.
+
+        Returns:
+            A list of experiment slugs.
+        """
+        matching = []
+        if self.excluded_experiments.exists():
```

Comment:
we don't include the docs, may you can add a line comment, as in other properties we don't define input and returns

Reply:
Sounds good! The doc for [live multifeature](https://github.com/mozilla/experimenter/pull/10210) matches already landed, but I will take it out of this one since we don't maintain those 👍

Reply:
Just because we don't have a lot of in-code documentation doesn't mean we shouldn't add it for new code.

If you put this information in docstrings, then IDE integrations will be able to show you it when you hover over a method. If you put that information in comments, that information is only visible if you go to look at the actual method source, which makes it less discoverable and less useful overall.